tests/sample.env

```diff
@@ -24,3 +24,6 @@ TEXT_CONTENT=Writing directly from string as read_file is not passed
 FILE_STORAGE_GCS='{"token": "/path/to/google/creds.json"}'
 FILE_STORAGE_MINIO='{"endpoint_url": "http://localhost:9000","key": "xxxx", "secret": "xxxx"}'
 FILE_STORAGE_LOCAL='{"auto_mkdir": True}'
+
+TEST_PERMANENT_STORAGE='{"provider": "gcs", "credentials": {"token": "/path/to/google/creds.json"}}'
```

Comment:
@gaya3-zipstack Was this committed by mistake or are we using it ass a reference to test?

Reply:
reference to test. Dos not contain any real creds.